src/tests/integration/D3MAave.t.sol

```diff
@@ -1061,6 +1061,86 @@ contract D3MAaveTest is DSSTest {
         vat.frob(ilk, address(address(d3mAavePool)), address(this), address(this), 0, -100); // Some small amount of dai repaid
 
         // We should be able to close out the vault completely even though ink and art do not match
-        _setRelBorrowTarget(0);
+        // _setRelBorrowTarget(0);
+        d3mAavePlan.file("bar", 0);
+
+        d3mHub.exec(ilk);
```

Comment:
Consider adding pool `ink` and `art` checks before and after this call.

Reply:
I've modified substantially this test, let me know if looks good now.